draft-ietf-quic-transport.md

```diff
@@ -1734,6 +1735,46 @@ Flow control is described in detail in {{flow-control}}, and congestion control
 is described in the companion document {{QUIC-RECOVERY}}.
 
 
+## Stream Prioritization
+
+Stream multiplexing has a significant effect on application performance if
+resources allocated to streams are correctly prioritized.  Experience with other
+multiplexed protocols, such as HTTP/2 {{?RFC7540}} shows that effective
+prioritization strategies have a significant impact on performance.
+
+QUIC does not provide frames for exchanging priotization information.  Instead
+it relies on receiving priority information from the application that uses QUIC.
+Protocols that use QUIC are able to define any prioritization scheme that suits
+their application semantics.  A protocol might define explicit messages for
+signaling priority, such as those defined in HTTP/2, it could define rules that
```

Comment:
I generally don't bother with a second citation for something when it appears in the same section.